server/models/Collective.js

```diff
@@ -680,6 +679,15 @@ export default function (Sequelize, DataTypes) {
             // Prevent collective creation
             throw new Error('Collective creation failed');
           }
+
+          // Set default platformFeePercent
+          if (!instance.platformFeePercent) {
```

Comment:
My understanding is that this will reset `platformFeePercent` if we try to create a collective with `platformFeePercent === 0`. Not a big deal because I suppose we will add this flag after the collective creation, but using `isNil` could have been better:

```es6
if (!isNil(instance.platformFeePercent)) {
```

Reply:
Yes, that's actually better.
I've implemented it here: https://github.com/opencollective/opencollective-api/pull/3583/files#diff-768438be827d00a869e92ca592010413R684.
Although I wrote it explicitly in plain JS.